contents/docs/product-analytics/sql.md

```diff
@@ -0,0 +1,293 @@
+---
+title: SQL (beta)
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+SQL insights enable you to directly access data in PostHog using SQL queries.
+
+> SQL queries in PostHog don’t require the trailing semi-colon (`;`) of traditional SQL queries.
+
+## SQL commands
+
+You create SQL queries out of a list of commands that shape what data, filters, aggregations, and ordering we want to see. 
+
+### SELECT
+
+Use `SELECT` to select data (usually columns, transformations, or aggregations) from one or more tables in the database.
+
+```sql
+SELECT *
+FROM events
+```
+
+Common values to select are `*` (representing all), `event`, `timestamp`, `properties`, and functions. Add the `DISTINCT` clause to `SELECT` commands to keep only unique rows in query results.
+
+### FROM
+
+Use `FROM` to select the database table to run the queries on. In PostHog, examples include `events`, `groups`, `raw_session_replay_events`, and more listed in the [data management database tab](https://app.posthog.com/data-management/database) in your PostHog instance.
+
+```sql
+SELECT session_id, min_first_timestamp, click_count 
+FROM raw_session_replay_events
+```
+
+You can join multiple tables together using the `LEFT JOIN` command which takes one table before the command and another after the command and combines them based on the join condition using the `ON` keyword.
+
+```sql
+SELECT events.event, persons.is_identified
+FROM events
+LEFT JOIN persons ON events.person_id = persons.id
+```
+
+### WHERE
+
+Use `WHERE` to filter rows based on specified conditions. These conditions can be:
+
+1. Comparison operators like `=`, `<`, or `≤`
```

Comment:
Just checking this is right: You should use `≤` instead of `<=` ?

Reply:
You're right, should be >=